packages/venia-ui/lib/components/Navigation/navHeader.js

```diff
@@ -17,13 +17,19 @@ class NavHeader extends Component {
     };
 
     render() {
-        const { classes, onBack, onClose, title } = this.props;
+        const { classes, onBack, onClose, title, isCategoryRoot } = this.props;
```

Comment:
Please also add `isCategoryRoot` to this component's `propTypes`.

Reply:
@supernova-at , I am in no need to send this prop type anymore because after taking latest pull from develop and rework according to the latest deployed changes I found that there is already a variable i.e. isTopLevel been send in navHeader.

So I have used the same. Please have a look and let me know in case if you have any feedback in this.

Thanks.